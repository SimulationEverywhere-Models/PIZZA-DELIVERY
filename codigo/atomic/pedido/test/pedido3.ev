00:00:01:000 Pedido_in 1
00:00:01:002 Pedido_in 2
00:00:01:004 Pedido_in 3
00:00:01:006 Pedido_in 4
00:00:01:008 Pedido_in 5
00:00:04:008 Pedido_in 6
00:00:05:000 Esperar_p 1
00:00:06:000 Pedido_in 7
00:00:07:000 Pedido_in 8
00:00:08:000 Listo_p 1
00:00:09:000 Pedido_in 9
00:00:10:000 Pedido_in 10
