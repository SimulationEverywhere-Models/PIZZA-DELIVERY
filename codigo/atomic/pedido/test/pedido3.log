Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para pedido(02)
Mensaje D / 00:00:00:000 / pedido(02) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:01:000 / Root(00) / pedido_in /      1.00000 para top(01)
Mensaje X / 00:00:01:000 / top(01) / pedido_in /      1.00000 para pedido(02)
Mensaje D / 00:00:01:000 / pedido(02) / 00:00:00:030 para top(01)
Mensaje D / 00:00:01:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje X / 00:00:01:002 / Root(00) / pedido_in /      2.00000 para top(01)
Mensaje X / 00:00:01:002 / top(01) / pedido_in /      2.00000 para pedido(02)
Mensaje D / 00:00:01:002 / pedido(02) / 00:00:00:028 para top(01)
Mensaje D / 00:00:01:002 / top(01) / 00:00:00:028 para Root(00)
Mensaje X / 00:00:01:004 / Root(00) / pedido_in /      3.00000 para top(01)
Mensaje X / 00:00:01:004 / top(01) / pedido_in /      3.00000 para pedido(02)
Mensaje D / 00:00:01:004 / pedido(02) / 00:00:00:026 para top(01)
Mensaje D / 00:00:01:004 / top(01) / 00:00:00:026 para Root(00)
Mensaje X / 00:00:01:006 / Root(00) / pedido_in /      4.00000 para top(01)
Mensaje X / 00:00:01:006 / top(01) / pedido_in /      4.00000 para pedido(02)
Mensaje D / 00:00:01:006 / pedido(02) / 00:00:00:024 para top(01)
Mensaje D / 00:00:01:006 / top(01) / 00:00:00:024 para Root(00)
Mensaje X / 00:00:01:008 / Root(00) / pedido_in /      5.00000 para top(01)
Mensaje X / 00:00:01:008 / top(01) / pedido_in /      5.00000 para pedido(02)
Mensaje D / 00:00:01:008 / pedido(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:01:008 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:01:008 / Root(00) para top(01)
Mensaje * / 00:00:01:008 / top(01) para pedido(02)
Mensaje Y / 00:00:01:008 / pedido(02) / parar_p /      1.00000 para top(01)
Mensaje D / 00:00:01:008 / pedido(02) / 00:00:00:022 para top(01)
Mensaje Y / 00:00:01:008 / top(01) / parar_p /      1.00000 para Root(00)
Mensaje D / 00:00:01:008 / top(01) / 00:00:00:022 para Root(00)
Mensaje * / 00:00:01:030 / Root(00) para top(01)
Mensaje * / 00:00:01:030 / top(01) para pedido(02)
Mensaje Y / 00:00:01:030 / pedido(02) / pedido_out /      1.00000 para top(01)
Mensaje D / 00:00:01:030 / pedido(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:01:030 / top(01) / pedido_out /      1.00000 para Root(00)
Mensaje D / 00:00:01:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:01:030 / Root(00) para top(01)
Mensaje * / 00:00:01:030 / top(01) para pedido(02)
Mensaje Y / 00:00:01:030 / pedido(02) / libre_p /      1.00000 para top(01)
Mensaje D / 00:00:01:030 / pedido(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:01:030 / top(01) / libre_p /      1.00000 para Root(00)
Mensaje D / 00:00:01:030 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:01:060 / Root(00) para top(01)
Mensaje * / 00:00:01:060 / top(01) para pedido(02)
Mensaje Y / 00:00:01:060 / pedido(02) / pedido_out /      2.00000 para top(01)
Mensaje D / 00:00:01:060 / pedido(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:01:060 / top(01) / pedido_out /      2.00000 para Root(00)
Mensaje D / 00:00:01:060 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:01:090 / Root(00) para top(01)
Mensaje * / 00:00:01:090 / top(01) para pedido(02)
Mensaje Y / 00:00:01:090 / pedido(02) / pedido_out /      3.00000 para top(01)
Mensaje D / 00:00:01:090 / pedido(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:01:090 / top(01) / pedido_out /      3.00000 para Root(00)
Mensaje D / 00:00:01:090 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:01:120 / Root(00) para top(01)
Mensaje * / 00:00:01:120 / top(01) para pedido(02)
Mensaje Y / 00:00:01:120 / pedido(02) / pedido_out /      4.00000 para top(01)
Mensaje D / 00:00:01:120 / pedido(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:01:120 / top(01) / pedido_out /      4.00000 para Root(00)
Mensaje D / 00:00:01:120 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:01:150 / Root(00) para top(01)
Mensaje * / 00:00:01:150 / top(01) para pedido(02)
Mensaje Y / 00:00:01:150 / pedido(02) / pedido_out /      5.00000 para top(01)
Mensaje D / 00:00:01:150 / pedido(02) / ... para top(01)
Mensaje Y / 00:00:01:150 / top(01) / pedido_out /      5.00000 para Root(00)
Mensaje D / 00:00:01:150 / top(01) / ... para Root(00)
Mensaje X / 00:00:04:008 / Root(00) / pedido_in /      6.00000 para top(01)
Mensaje X / 00:00:04:008 / top(01) / pedido_in /      6.00000 para pedido(02)
Mensaje D / 00:00:04:008 / pedido(02) / 00:00:00:030 para top(01)
Mensaje D / 00:00:04:008 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:04:038 / Root(00) para top(01)
Mensaje * / 00:00:04:038 / top(01) para pedido(02)
Mensaje Y / 00:00:04:038 / pedido(02) / pedido_out /      6.00000 para top(01)
Mensaje D / 00:00:04:038 / pedido(02) / ... para top(01)
Mensaje Y / 00:00:04:038 / top(01) / pedido_out /      6.00000 para Root(00)
Mensaje D / 00:00:04:038 / top(01) / ... para Root(00)
Mensaje X / 00:00:05:000 / Root(00) / esperar_p /      1.00000 para top(01)
Mensaje X / 00:00:05:000 / top(01) / esperar_p /      1.00000 para pedido(02)
Mensaje D / 00:00:05:000 / pedido(02) / ... para top(01)
Mensaje D / 00:00:05:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:06:000 / Root(00) / pedido_in /      7.00000 para top(01)
Mensaje X / 00:00:06:000 / top(01) / pedido_in /      7.00000 para pedido(02)
Mensaje D / 00:00:06:000 / pedido(02) / ... para top(01)
Mensaje D / 00:00:06:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:07:000 / Root(00) / pedido_in /      8.00000 para top(01)
Mensaje X / 00:00:07:000 / top(01) / pedido_in /      8.00000 para pedido(02)
Mensaje D / 00:00:07:000 / pedido(02) / ... para top(01)
Mensaje D / 00:00:07:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:08:000 / Root(00) / listo_p /      1.00000 para top(01)
Mensaje X / 00:00:08:000 / top(01) / listo_p /      1.00000 para pedido(02)
Mensaje D / 00:00:08:000 / pedido(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:08:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:08:000 / Root(00) para top(01)
Mensaje * / 00:00:08:000 / top(01) para pedido(02)
Mensaje Y / 00:00:08:000 / pedido(02) / pedido_out /      7.00000 para top(01)
Mensaje D / 00:00:08:000 / pedido(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:08:000 / top(01) / pedido_out /      7.00000 para Root(00)
Mensaje D / 00:00:08:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:08:030 / Root(00) para top(01)
Mensaje * / 00:00:08:030 / top(01) para pedido(02)
Mensaje Y / 00:00:08:030 / pedido(02) / pedido_out /      8.00000 para top(01)
Mensaje D / 00:00:08:030 / pedido(02) / ... para top(01)
Mensaje Y / 00:00:08:030 / top(01) / pedido_out /      8.00000 para Root(00)
Mensaje D / 00:00:08:030 / top(01) / ... para Root(00)
Mensaje X / 00:00:09:000 / Root(00) / pedido_in /      9.00000 para top(01)
Mensaje X / 00:00:09:000 / top(01) / pedido_in /      9.00000 para pedido(02)
Mensaje D / 00:00:09:000 / pedido(02) / 00:00:00:030 para top(01)
Mensaje D / 00:00:09:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:09:030 / Root(00) para top(01)
Mensaje * / 00:00:09:030 / top(01) para pedido(02)
Mensaje Y / 00:00:09:030 / pedido(02) / pedido_out /      9.00000 para top(01)
Mensaje D / 00:00:09:030 / pedido(02) / ... para top(01)
Mensaje Y / 00:00:09:030 / top(01) / pedido_out /      9.00000 para Root(00)
Mensaje D / 00:00:09:030 / top(01) / ... para Root(00)
Mensaje X / 00:00:10:000 / Root(00) / pedido_in /     10.00000 para top(01)
Mensaje X / 00:00:10:000 / top(01) / pedido_in /     10.00000 para pedido(02)
Mensaje D / 00:00:10:000 / pedido(02) / 00:00:00:030 para top(01)
Mensaje D / 00:00:10:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:10:030 / Root(00) para top(01)
Mensaje * / 00:00:10:030 / top(01) para pedido(02)
Mensaje Y / 00:00:10:030 / pedido(02) / pedido_out /     10.00000 para top(01)
Mensaje D / 00:00:10:030 / pedido(02) / ... para top(01)
Mensaje Y / 00:00:10:030 / top(01) / pedido_out /     10.00000 para Root(00)
Mensaje D / 00:00:10:030 / top(01) / ... para Root(00)
