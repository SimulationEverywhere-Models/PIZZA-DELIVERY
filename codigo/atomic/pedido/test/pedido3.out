00:00:01:008 parar_p 1
00:00:01:030 pedido_out 1
00:00:01:030 libre_p 1
00:00:01:060 pedido_out 2
00:00:01:090 pedido_out 3
00:00:01:120 pedido_out 4
00:00:01:150 pedido_out 5
00:00:04:038 pedido_out 6
00:00:08:000 pedido_out 7
00:00:08:030 pedido_out 8
00:00:09:030 pedido_out 9
00:00:10:030 pedido_out 10
